.github/workflows/check-publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Check publish
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  check-publish:
+    strategy:
+      matrix:
```

Comment:
Just out of curiosity. Why do the `matrix` is used?

Reply:
I copied it from the other check workflow. I'm not sure if it's a holdover from when there were more OS's. Thought I'd keep it just to be consistent.